packages/oc-server-compiler/test/__snapshots__/oc-webpack.test.js.snap

```diff
@@ -114,7 +114,7 @@ Object {
     "webpack/runtime/define property getters",
     "webpack/runtime/hasOwnProperty shorthand",
     "webpack/runtime/make namespace object",
-    "/mocks/jade-component/server.js",
+    "/mocks/jade-component/mocks/jade-component/server.js",
```

Comment:
something weird happened here?

Reply:
this is the sourcemap that webpack generates for itself, so its all internal (it just happens when upgrading to webpack 5), so I think we are fine